features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/customreaction/DefaultEmojibaseProvider.kt

```diff
@@ -14,16 +14,14 @@
  * limitations under the License.
  */
 
-package io.element.android.x.initializer
+package io.element.android.features.messages.impl.timeline.components.customreaction
 
-import androidx.startup.Initializer
-import com.vanniktech.emoji.EmojiManager
-import com.vanniktech.emoji.google.GoogleEmojiProvider
+import android.content.Context
+import io.element.android.emojibasebindings.EmojibaseDatasource
+import io.element.android.emojibasebindings.EmojibaseStore
 
-class EmojiInitializer : Initializer<Unit> {
-    override fun create(context: android.content.Context) {
-        EmojiManager.install(GoogleEmojiProvider())
+class DefaultEmojibaseProvider(val context: Context) :EmojibaseProvider {
+    override fun loadEmojibase(): EmojibaseStore {
+        return EmojibaseDatasource().load(context)
```

Comment:
Should this be lazy loaded into memory?

Reply:
I decided to load it each time as it was relatively quick vs having it held in memory when it's not being used.

You think it'd be better to hold in memory? So that subsequent loads are quicker?

Reply:
It depends on how much memory it uses while loaded, I guess, but it's true that it loads everything quite fast. 

Making this `DefaultEmojibaseProvider` a singleton and having `loadEmojiBase()` return some lazy loaded value would be more efficient as we'd only have to do it once, but if wastes too much memory it might not be worth it.

Reply:
Yea, on reflection, airing on the side of snappier UI is probably what we are going for now. A slight delay each time you go to leave a reaction might be annoying. Plus, this would bring it in line with iOS.